GOT-StudentVersion/GOT-StudentVersion/ViewController.swift

```diff
@@ -8,10 +8,163 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
+    
+    var episodes = GOTEpisode.allEpisodes
+    
```

Comment:
You are missing a variable that holds the value of the filtered array of array of GOT properties: 
var arrOfSeasons = [[GOTEpisode]]() {
        didSet{
            tableView.reloadData()
        }
    }
    
    private func filterSeason () -> [[GOTEpisode]] {
        arrOfSeasons = [GOTEpisode.allEpisodes.filter{$0.season == 1}, GOTEpisode.allEpisodes.filter{$0.season == 2},GOTEpisode.allEpisodes.filter{$0.season == 3},GOTEpisode.allEpisodes.filter{$0.season == 4},GOTEpisode.allEpisodes.filter{$0.season == 5},GOTEpisode.allEpisodes.filter{$0.season == 6}, GOTEpisode.allEpisodes.filter{$0.season == 7}]
        return arrOfSeasons
    }